pytheus/exceptions.py

```diff
@@ -1,6 +1,2 @@
 class PytheusException(Exception):
     pass
-
-
-class UnobservableMetricException(PytheusException):
```

Comment:
I move subclasses of exceptions down to prevent circular imports. Some of these exceptions might need to use objects from where they are raised.